src/test/java/seedu/task/logic/LogicManagerTest.java

```diff
@@ -394,6 +394,31 @@ public void execute_select_jumpsToCorrectTask() throws Exception {
     }
 
     @Test
+    public void execute_MarkInvalidArgsFormat_errorMessageShown() throws Exception {
```

Comment:
bad phrasing of previous questions. 
Did my merging follow what you wanted initially? 
Because there are conflicts, and both conflicts are what you have written. So i kind of have no clue which version u intended. 


If ok, I will do PR liao.